### ISSUE TITLE
Skip annocheck tests for specific files in YAML

### ISSUE THREAD

**Issue by Enchufa2** [rpminspect/rpminspect#1135]
**Is your feature request related to a problem? Please describe.**

annocheck [reports](https://artifacts.dev.testing-farm.io/874650e7-93aa-44df-aecc-46d57a4183e9/) a bunch of false positives (fortify and glibcxx-assertions) for FlexiBLAS. My understanding is that, for the latter, I can add a `rpminspect.yaml` file to the package repo as follows:

```yaml
annocheck:
    jobs:
        hardened: --ignore-unknown --skip-glibcxx-assertions
```

because the libraries being reported do not use C++. But I wouldn't like to disable the fortify check globally.

**Describe the solution you'd like**

I'm not sure if this is possible already (I didn't find anything in the docs or examples), but it would be nice to be able to skip a specific check for a particular file.

**Describe alternatives you've considered**

I've considered the syntax `–skip-name[=funcname]`, but there are too many functions to collect them manually. These functions follow a pattern, but to my knowledge, a glob or regexp doesn't work.

**Comment by Enchufa2** [comment]
Thanks, and is it possible to specify several of them as a list? E.g.

```yaml
annocheck:
    extra_opts:
        hardened:
            - --skip-test1
            - --skip-test2
```
